src/newSeed/stage4.html

```diff
@@ -104,7 +106,8 @@
                 class="vestingDateInput"
                 value.bind="endDate | date"
                 placeholder="YYYY-MM-DD"
-                ref="endDateRef" />
+                ref="endDateRef" 
+                autocomplete="off" disabled/>
```

Comment:
why disabled?

Reply:
To disallow users from inputting into the input field